recipes/python-avro/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.7.7" %}
+
+package:
+  name: python-avro
+  version: {{ version }}
+
+source:
+  # Avro exist as two versions in apache avro for py2 and py3.
+  fn: avro-{{ version }}.tar.gz  # [py27]
+  url: https://pypi.python.org/packages/source/a/avro/avro-{{ version }}.tar.gz  # [py27]
+  md5: e21ae1dffea4c96ff151ce35ca5a8875  # [py27]
+
+  fn: avro-python3-{{ version }}.tar.gz  # [py3k]
+  url: https://pypi.python.org/packages/source/a/avro-python3/avro-python3-{{ version }}.tar.gz  # [py3k]
+  md5: bd36c1e6fbb77ccff58c60bed48b982e  # [py3k]
```

Comment:
I know I asked this, but I think the reply got buried. Why do what Python 3 and Python 2 sources that are different?